src/Gaufrette/Adapter/AwsS3.php

```diff
@@ -111,10 +111,12 @@ public function rename($sourceKey, $targetKey)
 
         try {
             $this->service->copyObject($options);
-            return true;
         } catch (\Exception $e) {
             return false;
         }
+        $this->delete($sourceKey);
```

Comment:
I guess the question is what we do here, because obviously the behaviour should be as similar as possible.  I've changed the commit here to be logically the same as the v1 adapter - hopefully that's OK/ no worse than before?